Python/Product/Analyzer/Intellisense/AnalysisProtocol.cs

```diff
@@ -851,5 +851,50 @@ internal class UnhandledExceptionEvent : Event {
             }
             public override string name => Name;
         }
+
+        public sealed class GetTestCasesRequest : Request<GetTestCasesResponse> {
+            public const string Command = "getTestCases";
+
+            public override string command => Command;
+
+            public string filename;
```

Comment:
Guessing we can extend this easily/safely in the future? If, say, we wanted to add alternate detection methods other than `unittest`?